components/dashboard/src/components/Menu.tsx

```diff
@@ -39,22 +39,20 @@ function MenuItem(entry: Entry) {
     </li>;
 }
 
-function Menu(props: { left: Entry[], right: Entry[] }) {
+function Menu(props: { list: Entry[]}) {
     const { user } = useContext(UserContext);
 
     return (
         <header className="lg:px-28 px-10 bg-white flex flex-wrap items-center py-4">
             <div className="flex justify-between items-center pr-3">
                 <Link to="/">
-                    <img src={gitpodIcon} className="h-6" />
+                    <img src={getLogoUrl("brand")} className="h-10" />
                 </Link>
             </div>
             <div className="flex flex-1 items-center w-auto w-full" id="menu">
                 <nav className="flex-1">
-                    <ul className="flex flex-1 items-center justify-between text-base text-gray-700 space-x-2">
-                        {props.left.map(MenuItem)}
-                        <li className="flex-1"></li>
-                        {props.right.map(MenuItem)}
+                    <ul className="flex flex-1 items-center justify-center text-base text-gray-700 space-x-2">
```

Comment:
Try to keep the structure very similar to whats available; If its left and right and we want things in left, lets not rename to list.

Reply:
Okay.